packages/components/src/Tile/subcomponents/GenericTile/GenericTile.tsx

```diff
@@ -46,13 +49,23 @@ export const GenericTile = ({
   titleTag = "h3",
   metadata,
   information,
+  infoButtonLabel,
   mood,
   variant = "default",
   footer,
   classNameOverride,
   ...restProps
 }: GenericTileProps): JSX.Element => {
   const [isFlipped, setIsFlipped] = useState<boolean>(false)
+  const { formatMessage } = useIntl()
+
+  infoButtonLabel = infoButtonLabel
+    ? formatMessage({
```

Comment:
This one we'll want to flip the logic a little. When the `infoButtonLabel` is undefined we'll provide a translatable default, but when it is defined, we can assume that our users will provide a translated string.

There's also some small things we'll need to tweak on the description, but we can talk about that in our pairing catch up :)

Reply:
Done!